### ISSUE TITLE
fixity declarations don't seem to work

### ISSUE THREAD

**Issue by miguel-montes** [gibiansky/IHaskell#699]
If I try to declare associativity and precedence of an operator, it doesn't seem to have effect:

``` haskell
infixr 6 @@
(@@) = (-)

1 @@ 2 @@ 3 -- Should be equivalent to 1 - (2 - 3)
```

In ghci the result is 2, as expected, but with IHaskell is -4.

**Comment by leftaroundabout** [comment]
This seems to be a problem with recent IHaskell versions using GHC-7, but not with GHC-8 (which is all that's actually supported at the moment).

**Comment by leftaroundabout** [comment]
Workaround: put the infix definitions in a `module` cell, this way it will get treated just like any other Haskell file by GHC, instead of passing line-by-line through IHaskell. Hence it will also work in GHC-7.

But really, the “correct” fix is to switch to the supported GHC-8.0.

@vaibhavsagar close issue? This is certainly not reproducible with the recommended Stack setup.

**Comment by vaibhavsagar** [comment]
Sure, thanks!
